frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimePlatformsLabelGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import { Label, LabelGroup } from '@patternfly/react-core';
+import * as React from 'react';
+import { TemplateKind } from '~/k8sTypes';
+import { getEnabledPlatformsFromTemplate } from '~/pages/modelServing/customServingRuntimes/utils';
+import { ServingRuntimePlatform } from '~/types';
+
+type CustomServingRuntimePlatformsLabelGroupProps = {
+  template: TemplateKind;
+};
+
+const ServingRuntimePlatformLabels = {
+  [ServingRuntimePlatform.SINGLE]: 'Single model',
+  [ServingRuntimePlatform.MULTI]: 'Multi-model',
+};
+
+const CustomServingRuntimePlatformsLabelGroup: React.FC<
+  CustomServingRuntimePlatformsLabelGroupProps
+> = ({ template }) => {
+  const platforms = getEnabledPlatformsFromTemplate(template);
+
+  if (platforms.length === 0) {
+    return null;
+  }
+
+  return (
+    <LabelGroup>
+      {platforms.map((platform, i) => (
+        <Label key={i}>{ServingRuntimePlatformLabels[platform]}</Label>
```

Comment:
Small nit... it's not a huge thing usually to worry about this... but `key` has rendering value (mainly for performance). If you have rendered text like labels, you probably could just use that as the key. If that's not unique, the user will likely be confused too.

Unless we are rendering a ton of items, it typically won't matter one way or another. But I like to think it gives the developer more understanding of React to treat `key`s with the intent they have.